codewof/templates/users/dashboard.html

```diff
@@ -30,11 +30,23 @@ <h2>Your questions for today</h2>
             <strong>Sorry!</strong> We currently don't have questions for you to do today.
           </div>
         {% endif %}
+        <p>You've solved {{num_questions_answered}} question{{ num_questions_answered|pluralize }} in the last month!</p>
+<!--        {% include "users/activity_graph.html" %}-->
       </div>
 
       <div class="col-12 col-md-4 mb-3">
-        <h2>Achievements</h2>
-        <p><em>Coming soon!</em></p>
+        <h2>Points earned: {{user.profile.points}}</h2>
+        <h2><a id="achievements-link" href="{% url 'users:achievements' %}">Achievements</a></h2>
+        <br>
+        {% for badge in all_badges %}
+            {% if badge in user.profile.earned_badges.all and badge.parent not in user.profile.earned_badges.all %}
+              <div class="badge-container">
+                <img class="badge-icon" src="{% static badge.icon_name %}">
+                <b>{{badge.display_name}}</b>
+              </div>
+            {% endif %}
+<!--            {% include "programming/badge_card.html" %}-->
```

Comment:
What's this?

Reply:
I really don't know what it's supposed to be... should I delete the associated code?